apps/meteor/app/livechat/server/hooks/saveLastMessageToInquiry.ts

```diff
@@ -0,0 +1,35 @@
+import { isOmnichannelRoom } from '@rocket.chat/core-typings';
+
+import { callbacks } from '../../../../lib/callbacks';
+import { LivechatInquiry } from '../../../models/server/raw';
+import { settings } from '../../../settings/server';
+import { RoutingManager } from '../lib/RoutingManager';
+
+callbacks.add(
+	'afterSaveMessage',
+	async (message, room) => {
+		if (!isOmnichannelRoom(room)) {
+			return message;
+		}
+
+		// skip callback if message was edited
+		if ((message as any).editedAt) {
```

Comment:
pls avoid `any` as must as possible 🙈 
```suggestion
		if (message.editedAt) {
```

Reply:
in this case you could also use `isEditedMessage` from `IMessage.ts`

Reply:
Nice catch, let me change it 👀